README.md

```diff
@@ -25,9 +25,9 @@ Helper   | JavaScript                                      | HTMLBars
 eq       | `if (a === b)`                                  | `{{if (eq a b)}}`       | No                              |
 not-eq   | `if (a !== b)`                                  | `{{if (not-eq a b)}}`   | No                              |
 not      | `if (!a)`                                       | `{{if (not a)}}`        | Yes                             |
-and      | `if (a && b)`*                                  | `{{if (and a b)}}`      | Yes                             |
-or       | <code>if (a &#124;&#124; b)</code> *            | `{{if (or a b)}}`       | Yes                             |
-xor      | <code>if (a && !b &#124;&#124; !a && b)</code>* | `{{if (xor a b)}}`      | No                              |
+and      | `if (a && b)`                                   | `{{if (and a b)}}`      | Yes                             |
+or       | <code>if (a &#124;&#124; b)</code>              | `{{if (or a b)}}`       | Yes                             |
+xor      | <code>if (a !== b)</code>                       | `{{if (xor a b)}}`      | No                              |
```

Comment:
`or`, `and` do now short-circuit

for `xor` I felt writing it this way, how the helper is actually implemented, was clearer than needing to describe short-circuiting behaviour. It's clear that both a and b would be evaluated for this expression